tests/test_indexing.py

```diff
@@ -1037,6 +1037,114 @@ def test_fill_diagonal_error():
         dpnp.fill_diagonal(arr, 5)
 
 
+class TestRavelIndex:
+    def test_basic(self):
+        expected = numpy.ravel_multi_index(numpy.array([1, 0]), (2, 2))
+        result = dpnp.ravel_multi_index(dpnp.array([1, 0]), (2, 2))
+        assert_equal(expected, result)
+
+        x_np = numpy.array([[3, 6, 6], [4, 5, 1]])
+        x_dp = dpnp.array([[3, 6, 6], [4, 5, 1]])
+
+        expected = numpy.ravel_multi_index(x_np, (7, 6))
+        result = dpnp.ravel_multi_index(x_dp, (7, 6))
+        assert_equal(expected, result)
+
+    def test_mode(self):
+        x_np = numpy.array([[3, 6, 6], [4, 5, 1]])
+        x_dp = dpnp.array([[3, 6, 6], [4, 5, 1]])
+
+        expected = numpy.ravel_multi_index(x_np, (4, 6), mode="clip")
+        result = dpnp.ravel_multi_index(x_dp, (4, 6), mode="clip")
+        assert_equal(expected, result)
+
+        expected = numpy.ravel_multi_index(x_np, (4, 4), mode=("clip", "wrap"))
+        result = dpnp.ravel_multi_index(x_dp, (4, 4), mode=("clip", "wrap"))
+        assert_equal(expected, result)
+
+    def test_order_f(self):
+        x_np = numpy.array([[3, 6, 6], [4, 5, 1]])
+        x_dp = dpnp.array([[3, 6, 6], [4, 5, 1]])
+        expected = numpy.ravel_multi_index(x_np, (7, 6), order="F")
+        result = dpnp.ravel_multi_index(x_dp, (7, 6), order="F")
+        assert_equal(expected, result)
+
+    def test_error(self):
+        assert_raises(
+            ValueError, dpnp.ravel_multi_index, dpnp.array([2, 1]), (2, 2)
+        )
+        assert_raises(
+            ValueError, dpnp.ravel_multi_index, dpnp.array([0, -3]), (2, 2)
+        )
+        assert_raises(
+            ValueError, dpnp.ravel_multi_index, dpnp.array([0, 2]), (2, 2)
+        )
+        assert_raises(
+            TypeError, dpnp.ravel_multi_index, dpnp.array([0.1, 0.0]), (2, 2)
+        )
+
+    def test_empty_indices_error(self):
```

Comment:
The name of the test could be confusing (you can append them to the previous one), these tests raise Error not because of empty array. The first and second ones raise Error because they input is not a dpnp_array. The third one raise an Error because its dtype is float